config/routes.rb

```diff
@@ -219,7 +219,7 @@
           resources :webhook_endpoints, only: :index
         end
 
-        scope path: :app do
+        scope path: :apps do
```

Comment:
Do we need to account for any existing links to `app`?

Reply:
@jgaskins no this was actually a typo which led to a bug. However, because it was under a feature flag and a WIP it did not affect any users and just needed to be rectified as we make progress up this feature.